tests/platform_tests/test_advanced_reboot.py

```diff
@@ -130,3 +130,53 @@ def test_cancelled_warm_reboot(request, add_fail_step_to_reboot, verify_dut_heal
     add_fail_step_to_reboot('warm-reboot')
     advancedReboot = get_advanced_reboot(rebootType='warm-reboot', allow_fail=True)
     advancedReboot.runRebootTestcase()
+
+
+def test_service_warm_restart(request, duthosts, rand_one_dut_hostname, verify_dut_health, get_advanced_reboot,
```

Comment:
service_warm_restart is different than reboot cases handled in this script. I think it is better to create a different script for warm restart case(s).

Reply:
`test_service_warm_restart.py`?

Reply:
will fix